xml/System.Runtime.InteropServices/DllImportAttribute.xml

```diff
@@ -253,7 +253,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- You set this field to one of the <xref:System.Runtime.InteropServices.CallingConvention> enumeration members. The default value for the <xref:System.Runtime.InteropServices.DllImportAttribute.CallingConvention> field is <xref:System.Runtime.InteropServices.CallingConvention.Winapi>, which in turn defaults to <xref:System.Runtime.InteropServices.CallingConvention.StdCall> convention.  
+ You set this field to one of the <xref:System.Runtime.InteropServices.CallingConvention> enumeration members. The default value for the <xref:System.Runtime.InteropServices.DllImportAttribute.CallingConvention> field is <xref:System.Runtime.InteropServices.CallingConvention.Winapi>, which in turn defaults to <xref:System.Runtime.InteropServices.CallingConvention.StdCall> convention on Windows, and <xref:System.Runtime.InteropServices.CallingConvention.Cdecl> on Linux.
```

Comment:
Small change suggested.

~~on Linux.~~ -> on all other platforms.

Reply:
Thanks for suggestion. I'll go ahead and update the documentation on `CallingConvention` in a similar manner, too.

Reply:
> I'll go ahead and update the documentation on `CallingConvention` in a similar manner, too.

Decided not to do that, since it only talks about examples on x86, and it would be weird to talk about x86 examples on, say, macOS. So let it be as is.